source/aggregation-tutorials/group-total.txt

```diff
@@ -0,0 +1,225 @@
+.. _node-aggregation-group-total:
+
+===============
+Group and Total
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: code example, node.js, analyze, aggregation
+
+Introduction
+------------
+
+In this tutorial, you can learn how to use the {+driver-short+} to
+group documents and create totals from these groups. The aggregation
+groups documents on a common field and combines information
+from the documents to create new fields.
+
+Aggregation Task Summary
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+This aggregation tutorial demonstrates how to group and analyze data
+describing customer orders. The aggregated result is a list of customers
+who purchased items in 2020. The results include information about
+each customer's order history.
+
+This example uses one collection, ``orders``, which contains documents
+describing individual orders for products in a shop.
```

Comment:
Suggestion:
I think mentioning "shop" is unnecessary because it's not used in any other context. I also think that "orders for products" is redundant and would probably refer to them as "product purchases", but I think introducing terminology that differs from the field names might be confusing.

```suggestion
that describe product orders.
```